library-best-practices/JLBP-18.md

```diff
@@ -0,0 +1,49 @@
+[JLBP-18] Use shading sparingly and carefully
```

Comment:
Use shading --> shade

Reply:
I think that might be a little terse for people not familiar with shading. I'd prefer to keep the existing form.

Reply:
I think I'll go with "Only shade dependencies as a last resort". I think I want to avoid the command word at the beginning since we don't want anyone to fixate on the "shade" part that comes first and then think they need to do that (if they don't read carefully).